hack/rbacgenerator/function.js

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Hm, maybe, sure. I'm open to suggestions. It's not totally unprecedented, kubernetes has a hack/ dir for temp/dev type things.